core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -1484,40 +1330,40 @@ private ExprValue exprDateDiff(
   }
 
   /**
-   * DateTime implementation for ExprValue.
+   * Timestamp implementation for ExprValue.
    *
-   * @param dateTime ExprValue of String type.
+   * @param timestamp ExprValue of String type.
    * @param timeZone ExprValue of String type (or null).
    * @return ExprValue of date type.
    */
-  private ExprValue exprDateTime(ExprValue dateTime, ExprValue timeZone) {
+  private ExprValue exprDateTime(ExprValue timestamp, ExprValue timeZone) {
     String defaultTimeZone = TimeZone.getDefault().getID();
 
     try {
       LocalDateTime ldtFormatted =
-          LocalDateTime.parse(dateTime.stringValue(), DATE_TIME_FORMATTER_STRICT_WITH_TZ);
+          LocalDateTime.parse(timestamp.stringValue(), DATE_TIME_FORMATTER_STRICT_WITH_TZ);
       if (timeZone.isNull()) {
-        return new ExprDatetimeValue(ldtFormatted);
+        return new ExprTimestampValue(ldtFormatted);
       }
 
-      // Used if datetime field is invalid format.
+      // Used if timestamp field is invalid format.
     } catch (DateTimeParseException e) {
       return ExprNullValue.of();
     }
 
     ExprValue convertTZResult;
-    ExprDatetimeValue ldt;
+    ExprTimestampValue ldt;
```

Comment:
Change variable name since `ldt` probably stands for localdatetime

Reply:
changed variable name from `ldt` to `tz` in https://github.com/Bit-Quill/opensearch-project-sql/commit/f6edc6be62abbe3a3974665e8d51c515584234e0